modules/odo-push-watch-to-odo-dev.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/odo-migration-guide.adoc
+
+:_content-type: CONCEPT
+[id="odo-push-watch-to-odo-dev_{context}"]
+
+= odo push and odo watch replaced by odo dev
+
+`odo` v3 replaces the commands `odo push` and `odo watch` from v2 with a single command `odo dev`. 
+
+With `odo` v2, if you wanted to automatically synchronize the code running on the developer system with the application running on a Kubernetes cluster, you had to perform two steps:
```

Comment:
```suggestion
In `odo` v2, if you wanted to automatically synchronize the code running on the developer system with the one running on your cluster, you had to perform two steps:
```

I am not sure if it should be 'Kubernetes cluster' or 'OpenShift cluster' since this is a downstream doc, I am more inclined towards the latter.